spec/unit/helpers/check_helper_spec.rb

```diff
@@ -131,4 +131,41 @@
       ['upper_case_param', :upper_case_param, 'UPPER_case_param', :UPPER_case_param].map { |e| expect(params[e]).to eq('qux') }
     end
   end
+
+  context 'when params contain enum' do
+    before { params.setup_filters(spec) }
+
+    let(:spec) do
+      GoodData::LCM2::BaseAction.define_params(self) do
+        param :enum_type_param,
+        instance_of(GoodData::LCM2::Type::SynchronizeLDM)
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.